job-descriptions/software-engineer-template.md

```diff
@@ -0,0 +1,47 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Security
+
+_Describe the role in a few (~3-5) complete sentences. This should be inspiring to candidates and what someone might copy-and-paste when sharing or promoting a link to this job description._
+
+## Qualifications
+
+We are looking for candidates with the following skills, and we will test for these things in our interview process:
+
+- _Short list of qualifitications._
+
+## Nice-to-haves
+
+- _Short list of things that would be nice for a candidate to have, but aren't required._
+
+## Learn more about us
```

Comment:
I like that you moved this lower and that we're linking to other places that talk about Sourcegraph. That first section has been the most compelling to candidates I've had coming through my pipelines. One concern would be if the candidate doesn't want to take the time to do reading elsewhere, they won't get information about why Sourcegraph is awesome. However, if they have gotten this far in the job description they are likely already interested and will be willing to investigate.

Reply:
>That first section has been the most compelling to candidates I've had coming through my pipelines.

To be clear, the new first section, right?

> One concern would be if the candidate doesn't want to take the time to do reading elsewhere, they won't get information about why Sourcegraph is awesome.

The previous blurb didn't explain _that_ much and even if it is inline we can't force anyone to actually read it. Today we still get candidates who don't know much about Sourcegraph, so I think the difference between inlining or not is marginal (it is more a function of the candidate's interest and personality).

@keegancsmith said that he tends to not read the "Silicon Valley Speak" that we had in our previous About Us section and suggested exactly this ordering change: https://github.com/sourcegraph/careers/pull/61#discussion_r353744832

Reply:
Yes, the new first section. And yep, agree with the decision to move the content!